our-process/policies.md

```diff
@@ -77,6 +78,33 @@ a conflict of interest if:
 In the case where none of the associate editors can serve as editor, an
 external guest editor will be recruited to lead the package review.
 
+(generative-ai-policy)=
+## Policy for use of generative AI / LLMs
+
+:::{admonition} How this policy was developed
+:class: important
+
+The policy below was co-developed by the pyOpenSci community. Its goals are:
+
+* Acknowledgment of and transparency around the widespread use of Generative AI tools (with a focus on Large Language Models (LLMs) in Open Source.
```

Comment:
along with this - allow reviewers to make informed decisions about what they choose to review, and allow authors to have reviewers that align with their values and practices. facilitate a review process that aligns with the ethics and values of its participants.